recipe/meta.yaml

```
@@ -1,88 +1,77 @@
+{% set name = "ucx" %}
 {% set version = "1.18.1" %}
 
 package:
-  name: ucx-split
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/openucx/ucx/releases/download/v{{ version }}/ucx-{{ version }}.tar.gz
+  url: https://github.com/open{{ name }}/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
   sha256: 8018dd75f11b5e8d6e57dcdb5b798d2c1f000982c353efde1f3170025c6c3b4c
 
 build:
-  skip: true  # [(not linux) or cuda_compiler_version in (undefined, "None", "12.6")]
-  number: 2
+  number: 0
+  skip: True  # [(not linux) or cuda_compiler_version in (undefined, "None", "12.6")]
+  ignore_run_exports_from:
+    - {{ compiler("cuda") }}
+    - cuda-cudart-dev  # [cuda_compiler == "cuda-nvcc"]
+  missing_dso_whitelist:
+    - "*libcuda.so*"
+    - "*libcudart.so*"
+    - "*libnvidia-ml.so*"
+  run_exports:
+    - {{ pin_subpackage("ucx", max_pin="x.x") }}
 
-outputs:
-  - name: ucx
-    version: {{ version }}
-    build:
-      run_exports:
-        - {{ pin_subpackage("ucx", max_pin="x.x") }}
-      ignore_run_exports_from:
-        - {{ compiler("cuda") }}
-        - cuda-cudart-dev         # [cuda_compiler == "cuda-nvcc"]
-      missing_dso_whitelist:
-        - "*libcuda.so*"
-        - "*libcudart.so*"
-        - "*libnvidia-ml.so*"
-    requirements:
-      build:
-        - {{ stdlib("c") }}
-        - {{ compiler("c") }}
-        - {{ compiler("cxx") }}
-        - {{ compiler("cuda") }}
-        - automake
-        - autoconf
-        - libtool
-        - make
-        - pkg-config
-        - libgomp
-      host:
-        - cuda-version {{ cuda_compiler_version }}
-        - cuda-cudart-dev                                           # [cuda_compiler == "cuda-nvcc"]
-        - cuda-nvml-dev                                             # [cuda_compiler == "cuda-nvcc"]
-        - rdma-core
-      run_constrained:
-        - {{ pin_compatible("cuda-version", lower_bound="11.2") }}  # [cuda_compiler == "nvcc"]
-        - {{ pin_compatible("cuda-version", min_pin="x") }}         # [cuda_compiler == "cuda-nvcc"]
-        # The actual version constraint comes from cuda-version, so below we explicit list
-        # them only to codify that these are expected (but optional) dependencies when CUDA
-        # is in use.
-        - cudatoolkit                                               # [cuda_compiler == "nvcc"]
-        - cuda-cudart                                               # [cuda_compiler == "cuda-nvcc"]
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ compiler("cuda") }}
+    - automake
+    - autoconf
+    - libtool
+    - make
+    - pkg-config
+    - libgomp
+  host:
+    - cuda-version 12.4
```

```suggestion
    - cuda-version {{ cuda_compiler_version }}
```
You can rely on cbc.yaml https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L69